app/views/layouts/email.html.haml

```diff
@@ -0,0 +1,23 @@
+!!!
+%html
+  %head
+    %title C2
+    %meta{:content => "text/html; charset=US-ASCII", "http-equiv" => "Content-Type"}/
+    %meta{:content => "width=device-width", :name => "viewport"}/
+    %link{:href => stylesheet_url("mailer_new.css"), :rel => "stylesheet", :type => "text/css"}/
```

Comment:
do we still want to require this stylesheet? or can we remove this line and the stylesheet?